triphouse/index.html

```diff
@@ -6,6 +6,54 @@
     <link href="style.css" rel="stylesheet">
 </head>
 <body>
+<div class="top-section">
+
+    <div class="header-triphouse">
+        <img class="logo" src="images/logo-vector.svg" alt="logo">
+        <div class="header-triphouse-right">
+            <div>
+                <div class="header-triphouse-right"> Stays</div>
+                <div class="highlight-stays"></div>
+            </div>
+            <div class="header-triphouse-right"> Attraction</div>
+            <div class="header-triphouse-right">
+                <img src="images/night.svg" alt="night">
+            </div>
+            <div class="header-triphouse-right">
+                <img src="images/account-circle.svg" alt="account">
+            </div>
+        </div>
+    </div>
+    <div id="top-section-text">
+        Discover stays <br>
+        to live, work or just relax
+    </div>
+
+    <form class="form" method="get">
+        <div>
+            <label for="destination"> Your destination or hotel name</label> <br>
+            <input class="destination" id="destination" name="destination" type="text" placeholder="New York"> <br>
+        </div>
+        <div>
+            <label for="date"> Check-in &#8211; Check-out </label> <br>
+            <input class="date" id="date" name="date" type="date">
```

Comment:
@AlKurlk I'm sure that here you should set 2 input because this form point means the booking period